### PR TITLE
Update documentation to reflect 7.41.0 changing the default collectio…

### PR DESCRIPTION
**Pull request by alexandre-normand**
…n for the postgres database

### What does this PR do?

This updates a troubleshooting entry for postgres to note that we updated the [default collection configuration in agent 7.41.0](https://github.com/DataDog/integrations-core/pull/12999) (not released yet). 

### Additional Notes

https://docs-staging.datadoghq.com/alex.normand/update-postgres-doc-mentionning-ignore-databases/database_monitoring/setup_postgres/troubleshooting

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/database_monitoring/setup_postgres/troubleshooting.md [hunk 1]
```diff
@@ -182,7 +182,9 @@ If a client is using the Postgres [extended query protocol][9] or prepared state
...
 
 #### Query is in a database ignored by the Agent instance config
-The query is in a database ignored by the Agent instance config `ignore_databases`. Default databases such as the `postgres` database are ignored in the `ignore_databases` setting. Queries in these databases do not have samples or explain plans. Check the the value of this setting in your instance config and the default values in the [example config file][19].
+The query is in a database ignored by the Agent instance config `ignore_databases`. Default databases such as the `rdsadmin` and the `azure_maintenance` databases are ignored in the `ignore_databases` setting. Queries in these databases do not have samples or explain plans. Check the the value of this setting in your instance config and the default values in the [example config file][19].
+
+_Prior to Agent 7.41.0, the `postgres` database also ignored by default._
```
Comment (maycmlee) [inline content/en/database_monitoring/setup_postgres/troubleshooting.md:187]:
```suggestion
**Note:** The `postgres` database is also ignored by default in Agent versions <7.41.0.
```

**Review by maycmlee — COMMENTED** [review]
Left a suggestion!

**Comment by alexandre-normand** [comment]
> Left a suggestion!

I like it and committed your suggestion. Better than what I had. Thanks, @maycmlee !

**Review by maycmlee — APPROVED** [review]
Hey @alexandre-normand! Let me know if I should go ahead and merge this.

**Comment by alexandre-normand** [comment]
> Hey @alexandre-normand! Let me know if I should go ahead and merge this.

Yeah, I think that we can. 7.41.0 won't be out for a while but the documentation is still accurate.

**Pull request merged**